_config.yml

```diff
@@ -55,3 +55,6 @@ algolia:
     - assets
     - public
     - stylesheets
+
+sass:
+  quiet_deps: true
```

Comment:
When I ran it locally there were some deprecations with `lighten()` being used. This deprecation is a very recent development in the sass engine. So to silence deprecation warnings for the current version and all future versions I added this.